libs/platform/src/lib/components/table/table.component.ts

```diff
@@ -244,9 +244,9 @@ export class TableComponent<T = any> extends Table implements AfterViewInit, OnD
     @Input()
     relationKey: string;
 
-    /** Whether table row has navigation button. */
+    /** Whether row is navigatable. */
     @Input()
-    navigationButton = false;
+    rowNavigatable: string | boolean;
```

Comment:
It's not entirely clear what can come as a string? Which options of the string?

Reply:
https://github.com/SAP/fundamental-ngx/pull/6357#discussion_r692745286